app/assets/stylesheets/projects.css.scss

```diff
@@ -27,3 +27,21 @@ h5 span.favorite {
 .project_links {
   margin-bottom: 6%;
 }
+
+.tag_list {
+  border-radius: 5px;
+  color: #fff;
+  padding: 10px;
+  margin-bottom: 0;
+  overflow-x: scroll;
+  overflow-y: hidden;
+  white-space: nowrap;
+  
+  div {
+    display: inline-block;
+    padding-right: 10px;
+  }
+}
+
+.purple { background-color: $cmPurple; }
+.teal   { background-color: $cmTeal; }
```

Comment:
Files should end with a trailing newline<br>Name of variable `cmTeal` should be written in all lowercase letters with hyphens instead of underscores<br>Opening curly brace `{` should be preceded by one space